### ISSUE TITLE
[Bug]: scalafmt.conf includePaths/excludePaths ignored

### ISSUE THREAD

**Issue by vinnybod** [aspect-build/rules_lint#371]
### What happened?

I have a reproduction of the issue here: https://github.com/vinnybod/bazel_java_example/tree/rules_lint_config

TL;DR
I have a scalafmt.conf that specifies which files to format. When running directly via `scala-cli fmt`, it formats the correct files. When running with `bazel run format`, it formats every file.
```
project {
    includePaths = [
        "glob:**/app/**/FormatThis.scala",
    ]
    excludePaths = [
        "glob:**/app/**/DontFormatThis.scala"
    ]
}
```

### Version

Development (host) and target OS/architectures: MacOS

Output of `bazel --version`: 7.2.1

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:

Language(s) and/or frameworks involved:
scala - scalafmt

### How to reproduce
https://github.com/vinnybod/bazel_java_example/tree/rules_lint_config

Run `bazel test //tools/format:format_test --test_output=errors` to see that both `DontFormatThis.scala` and `FormatThis.scala`
 are attempted to be formatted.
 
```shell
==================== Test output for //tools/format:format_test_Scala_with_scalafmt:
--- a/Users/vinnybod/dev/vinnybod/bazel_java_example/app/src/main/scala/com/example/DontFormatThis.scala
+++ b/Users/vinnybod/dev/vinnybod/bazel_java_example/app/src/main/scala/com/example/DontFormatThis.scala
@@ -4,6 +4,4 @@
 
-
-  def main
-(args:
-Array[String]) = println("Hello, world")
+  def main(args: Array[String]) = println("Hello, world")
 }
+
--- a/Users/vinnybod/dev/vinnybod/bazel_java_example/app/src/main/scala/com/example/FormatThis.scala
+++ b/Users/vinnybod/dev/vinnybod/bazel_java_example/app/src/main/scala/com/example/FormatThis.scala
@@ -4,6 +4,4 @@
 
-
-  def main
-(args:
-Array[String]) = println("Hello, world")
+  def main(args: Array[String]) = println("Hello, world")
 }
+
error: --test failed
FAILED: A formatter tool exited with code 1
Try running 'bazel run //tools/format:format_test_Scala_with_scalafmt ' to fix this.
================================================================================
Target //tools/format:format_test_Scala_with_scalafmt up-to-date:
  bazel-bin/tools/format/format_test_Scala_with_scalafmt
INFO: Elapsed time: 1.241s, Critical Path: 1.13s
INFO: 2 processes: 2 local.
INFO: Build completed, 1 test FAILED, 2 total actions
```

Run `scala-cli fmt . --check` (I installed it via [coursier](https://get-coursier.io/docs/cli-installation#macos)).
See that only `FormatThis.scala` is attempted to be formatted.

```shell
--- a/Users/vinnybod/dev/vinnybod/bazel_java_example/app/src/main/scala/com/example/FormatThis.scala
+++ b/Users/vinnybod/dev/vinnybod/bazel_java_example/app/src/main/scala/com/example/FormatThis.scala
@@ -4,6 +4,4 @@
 
-
-  def main
-(args:
-Array[String]) = println("Hello, world")
+  def main(args: Array[String]) = println("Hello, world")
 }
+
error: --test failed
```

### Any other information?

I tested changing the maxColumns to make sure it picks up the config and it does, its just those paths that are not working. I wonder if it has something to do with the cwd when running through bazel.

**Comment by alexeagle** [comment]
FIxed by #372
